01-ESP32_firmware/include/main.hpp

```diff
@@ -11,32 +11,49 @@
 #define LED 2 //!< Onboard LED for signalizing
 
 /**
- * 
+ *  @brief Use it to initialize variables, pin modes, start using libraries, etc. 
+ * The setup() function will only run once
  */
-void setup();
+void setup(){
+
+}
 
 /**
- * 
+ * @brief Use it to actively control the board.
+ * After creating a setup() function, which initializes and sets the initial values, 
+ * the loop() function does precisely what its name suggests, and loops consecutively, 
+ * allowing your program to change and respond.
  */
-void loop();
+void loop(){
+
+}
 
 /**
- * 
+ * @brief  Function passed into another function as an argument, 
+ * Is then invoked inside the outer function to complete some kind of routine or action.
+ * @param topic
```

Comment:
@param topic mqtt topic of the received message

@param payload message body, e.g. text or JSON

@param length size of payload in bytes